epersons.md

```diff
@@ -2,10 +2,54 @@
 [Back to the list of all defined endpoints](endpoints.md)
 
 ## Main Endpoint
-**/api/eperson/epersons**  
+**GET /api/eperson/epersons**
 
 ## Single EPerson
-**/api/eperson/epersons/<:uuid>**
+**GET /api/eperson/epersons/<:uuid>**
+
+```json
+{
+  "id": "028dcbb8-0da2-4122-a0ea-254be49ca107",
+  "uuid": "028dcbb8-0da2-4122-a0ea-254be49ca107",
+  "name": "user@institution.edu",
+  "handle": null,
+  "metadata": [
+    {
+      "key": "eperson.firstname",
+      "value": "John",
+      "language": null
+    },
+    {
+      "key": "eperson.lastname",
+      "value": "Doe",
+      "language": null
+    },
+    {
+      "key": "eperson.phone",
+      "value": "",
+      "language": null
+    },
+    {
+      "key": "eperson.language",
+      "value": "en",
+      "language": null
+    }
+  ],
+  "netid": null,
+  "lastActive": null,
+  "canLogIn": true,
+  "email": "user@institution.edu",
+  "requireCertificate": false,
+  "selfRegistered": true,
+  "groups": null,
```

Comment:
Is this a list of group names or a href-list?

Reply:
This was replaced with a HAL link